joomla/joomla-cms
# Create custom admin article menus

See #20233 for detailed explanation of the use case

### Test Instructions

Create and enable an admin menu with the "Joomla preset"

Create a new admin menu item of type article list and you can see you now have a full set of filters that you can apply.

For example you can now create menu items in the admin to each category.

If approved I will create the same option for other components

This does not resolve everything on #20233 as it still needs field categories to be implemented for that

Thanks to @tonypartridge for his help
